src/lib/Components/Inbox/Conversations/index.tsx

```diff
@@ -111,10 +111,13 @@ export class Conversations extends Component<Props, State> {
           keyExtractor={(_item, index) => String(index)}
           scrollEventThrottle={500}
           renderItem={({ item }) => {
+            // TODO: What is up with this `as any`?
+            //       there's a type mismatch here, as item above is a `ConversationSnippetProps`
+            //       but conversation is a `ConversationSnippet_conversation`
```

Comment:
[This](https://github.com/artsy/emission/blob/535d3593f5a01641bed84f2281861b26768fb308/src/lib/Components/Inbox/Conversations/index.tsx#L30) should be:

```ts
interface State {
  conversations: Array<Props["me"]["conversations"]["edges"][0]["node"]> | null
}
```

(That could be simplified by making a separate fragment/container for the individual (conversation) nodes, which would give you a new type.)

And `me` should get typed [here](https://github.com/artsy/emission/blob/535d3593f5a01641bed84f2281861b26768fb308/src/lib/Components/Inbox/Conversations/index.tsx#L52)

Reply:
Btw, in that example `Props["me"]` is equal to `Conversations_me`, I go back and forth on what way to use 🤔